sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Assessments"
+        :routeLocation="goBackRouteParams"
+        subTitle="View Request"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestDetails.status ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithSABC
+          "
+        >
+          <v-row class="p-0 m-0">
```

Comment:
Checking other similar `header-navigator` template `#buttons` it seems that the `v-row` is not needed.
I noticed that it is added in many places, but can you please check if it is required?

https://github.com/bcgov/SIMS/blob/231107bfa6570abbd6d293a75679bc348c7d8e69/sources/packages/web/src/views/student/application-offering-change-request/ApplicationOfferingChangeFormView.vue#L9-L28

Reply:
Found it in two places around the code files I have worked on. Removed it from both the places.